feature_tests/features/consumer/consumer-countDocumentPointer-failure.feature

```diff
@@ -111,39 +111,12 @@ Feature: Consumer Search Failure scenarios
     Then the operation is unsuccessful
     And the status is 400
     And the response is an OperationOutcome according to the OUTCOME template with the below values
-      | property          | value                                                                 |
-      | issue_type        | processing                                                            |
-      | issue_level       | error                                                                 |
-      | issue_code        | VALIDATION_ERROR                                                      |
-      | issue_description | A parameter or value has resulted in a validation error               |
-      | message           | ConsumerRequestParams validation failure - Invalid subject:identifier |
-
-  Scenario: Search rejects request with type system they are not allowed to use
```

Comment:
Is there a reason why this scenario has been removed?

Reply:
Type is no longer accepted as a parameter